### PR TITLE
Update EyeDa App ReadMe

### PR DESCRIPTION
**Pull request by AbhiN2001**
Please review and let me know of any spelling mistakes!

### CODE REVIEW

**Review by YashDagade — APPROVED** [review]
Me don cee any selling mikstakes, does anyone else

File: EyeDa App/Readme.txt [hunk 1]
```diff
@@ -1 +1 @@
-For the Internship!
\ No newline at end of file
+This is the diectory wear the code for the EyeDaa V2 App will be stored! Done by Shreya Dixit Interens!
```
Comment (AbhiN2001) [inline EyeDa App/Readme.txt:1]: wear should be spelt where

**Review by YashDagade — CHANGES_REQUESTED** [review]
Change  a few other spellings

**Review by YashDagade — APPROVED** [review]

File: EyeDa App/Readme.txt [hunk 2]
```diff
@@ -1 +1 @@
-For the Internship!
\ No newline at end of file
+This is the diectory where the code for the EyeDaa V2 App will be stored! Done by Shreya Dixit Interens!
```
Comment (amar-gupta-dani) [inline EyeDa App/Readme.txt]: diectory should be spelled directory

**Review by amar-gupta-dani — APPROVED** [review]

**Review by nikhilopg4 — APPROVED** [review]

**Pull request merged**